pyvista/plotting/plotting.py

```diff
@@ -1318,7 +1318,21 @@ def camera_position(self, camera_location):
 
     @property
     def background_color(self):
-        """Return the background color of the active render window."""
+        """Return the background color of the active render window.
+
+        Examples
+        --------
+        Set the background color to ``"pink"`` and plot it.
+
+        >>> import pyvista as pv
+        >>> pl = pv.Plotter()
+        >>> _ = pl.add_mesh(pv.Cube(), show_edges=True)
+        >>> pl.background_color = "pink"
+        >>> pl.background_color
+        Color(name='pink', hex='#ffc0cbff')
```

Comment:
I want the feature to show the Color object in color bar. But it is other feature request. I'll create a issue.
```suggestion
        >>> pl.background_color
        Color(name='pink', hex='#ffc0cbff')
```
![image](https://user-images.githubusercontent.com/7513610/179425013-92346d1a-2266-487b-9f45-a265be102e4b.png)